src/components/QuranReader/TranslationView/TranslationText/index.tsx

```diff
@@ -155,11 +162,20 @@ const TranslationText: React.FC<Props> = ({
   const shouldShowFootnote = showFootnote && (footnote || isLoading);
   return (
     <div className={styles[`translation-font-size-${translationFontScale}`]} translate="no">
-      <div
-        onClick={(event) => onTextClicked(event)}
-        className={classNames(styles.text, styles[langData.direction], styles[langData.font])}
-        dangerouslySetInnerHTML={{ __html: text }}
-      />
+      {chapterName && reference ? (
+        <div className={classNames(styles.text, styles[langData.font])}>
+          &quot;{text}&quot;{' '}
```

Comment:
We should still use `dangerouslySetInnerHTML={{ __html: text }}` for text as before as the text from backend can be html! Also instead of ` &quot;` why don't we do {`"${text}"`}